saleor/graphql/attribute/types.py

```diff
@@ -128,12 +128,19 @@ class Meta:
 
     @staticmethod
     @traced_resolver
-    def resolve_values(root: models.Attribute, info, **_kwargs):
+    def resolve_choices(root: models.Attribute, _info, **_kwargs):
+        types_without_choices = [
+            AttributeInputType.REFERENCE,
+            AttributeInputType.NUMERIC,
+            AttributeInputType.RICH_TEXT,
+        ]
```

Comment:
This could be extracted to a constant and moved to `saleor/attribute/__init__.py`. This list may change only when adding a new input type (which happens in this init file), so it's good to have the two things as close to each other as possible.